perses/app/experiments.py

```diff
@@ -656,6 +657,9 @@ def _parse_ligand_input(self):
                 [oemol.SetTitle("MOL") for oemol in oemols]
                 self.smiles_list = [ oechem.OECreateSmiString(oemol, oechem.OESMILESFlag_DEFAULT | oechem.OESMILESFlag_Hydrogens) for oemol in oemols]
                 self.ligand_oemol_pos_top = [[oemol, position, forcefield_generators.generateTopologyFromOEMol(oemol)] for oemol, position in zip(oemols, positions)]
+                # Create openforcefield Molecule objects
+                from openforcefield.topology import Molecule
+                self.molecules = [ Molecule.from_openeye(oemol) for oemol in oemols ]
```

Comment:
is this just for the purposes of caching the molecules?

Reply:
because i think that the vacuum, solvated, and complex systems use the existing pipeline

Reply:
Currently, we only need the `openforcefield.topology.Molecule` objects for the `SystemGenerator` to know what to match inside the topologies it sees. I cached it within this object for convenience, but as long as we can get it to the `SystemGenerator`, it doesn't have to be cached.

Reply:
Eventually, I'd love to move away from all of these other objects (OEMol, positions, OpenMM Topology) and just use `openforcefield` `Molecule`s since we can generate all of those things from `Molecule`.

Reply:
> Currently, we only need the `openforcefield.topology.Molecule` objects for the `SystemGenerator` to know what to match inside the topologies it sees. I cached it within this object for convenience, but as long as we can get it to the `SystemGenerator`, it doesn't have to be cached.

ah right.  i understand now!